foreman-installer/foreman-installer.spec

```diff
@@ -1,12 +1,12 @@
 # set and uncomment all three to set alpha tag
-#global alphatag RC2
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+#%global alphatag
+#%global dotalphatag .%{alphatag}
+#%global dashalphatag -%{alphatag}
```

Comment:
Ah ha! This is the problem -- you can't comment out a definition directly. RPM spec will still process this. Thats why before it had dropped the `%` and I believe is the cause of the issue.